helix-view/src/editor.rs

```diff
@@ -117,7 +117,7 @@ impl Editor {
 
     pub fn new_file(&mut self, action: Action) -> DocumentId {
         use helix_core::Rope;
-        let doc = Document::new(Rope::from("\n"));
+        let doc = Document::new(Rope::new());
```

Comment:
See https://github.com/helix-editor/helix/pull/45/files#r644779412

Reply:
I'm not completely sure what is meant by this, but I did encounter a bug where it would panic because of `fragment()` or something like that. It's been fixed.